exercises/leap/cases_test.go

```diff
@@ -1,7 +1,7 @@
 package leap
 
 // Source: exercism/x-common
-// Commit: be6fa53 leap: Rewrite the test cases and their descriptions
+// x-common version: 1.0.0
```

Comment:
Well, there are advantages and disadvantages.

Advantage of commit: We'll know if there's **any** change, even if someone forgot to change the version when test contents change (I think x-common reviewers have been good at reminding people to change version though)

Advantage of version: We know how important the change is, if there is one - for example in a patch-level change (the last number) it isn't strictly necessary to rerun the generator since no test content will change.